AsyncDisplayKit/ASDisplayNode.h

```diff
@@ -185,6 +185,14 @@ typedef CALayer *(^ASDisplayNodeLayerBlock)();
  */
 @property (nonatomic, readonly, assign) ASSizeRange constrainedSizeForCalculatedLayout;
 
+/**
+ * @abstract Used by some implementations of measureWithSizeRange: to provide an intrisic content size
+ * to a node when one cannot be computed from its subnodes
```

Comment:
As a developer, I literally can't understand what this means in practice.  Consider any improvements to the language here, but I think it would be clear if an example is provided -- add a sentence describing the applicability to images in a grid.  You might say "For example, ASImageNode uses this property to define a default size for itself, without a developer needing to provide a custom layout spec"